src/components/InventoryGroups/GetHelpExpandable.js

```diff
@@ -32,7 +32,7 @@ const GetHelpExpandable = () => {
             className="ins-c-groups-help-expandable__link"
             size="lg"
             onClick={() =>
-              quickStarts.activateQuickstart('insights-inventory-groups')
+              quickStarts.activateQuickstart('insights-inventory-workspace')
```

Comment:
Should we put this behind a feature flag?

Reply:
@mkholjuraev agree, done